hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ListIterator.java

```diff
@@ -303,11 +323,12 @@ public static class MinHeapIterator implements ClosableIterator {
         for (Table table : tables) {
           CacheIter cacheIter = new CacheIter<>(iteratorId, table.getName(),
               table.cacheIterator(), startKey, prefixKey);
-          Predicate<String> isKeyExistInCache = cacheIter::isKeyExistInCache;
+          Predicate<String> doesKeyExistInCache =
```

Comment:
nit: JFYI, now char limit is 120. It can be fit in one line. Configure/update your IDE with latest ozone-style.

Reply:
ok done.